sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -29,9 +39,15 @@
     // Environment variable name used to determine the TestMode.
     private static final String AZURE_TEST_MODE = "AZURE_TEST_MODE";
     private static final String AZURE_TEST_HTTP_CLIENTS = "AZURE_TEST_HTTP_CLIENTS";
-    public static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ALL = "ALL";
-    public static final String AZURE_TEST_HTTP_CLIENTS_VALUE_NETTY = "NettyAsyncHttpClient";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ALL = "ALL";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ROLLING = "rolling";
+    private static final String AZURE_TEST_HTTP_CLIENTS_VALUE_NETTY = "NettyAsyncHttpClient";
```

Comment:
Should this just be 'netty'

Reply:
Yes. We can change to "netty". Will address this.